src/app/core/components/generic-mod-list/generic-mod-list.component.scss

```diff
@@ -40,13 +40,19 @@ mat-card {
 
 .mod-name {
   justify-content: center;
-  padding-top: 8px;
+  padding-top: 4px;
   padding-left: 8px;
   padding-right: 8px;
-  margin-bottom: 8px;
+  padding-bottom: 2px;
   min-width: 140px;
 }
 
+.mod-version {
+  padding: 0 8px;
+  padding-bottom: 2px;
```

Comment:
I would remove it to save some space. If you resize the window smaller, the button (to add remove, etc) will be displayed outside of the card

Reply:
With the smaller text size it's fine, the spacer above the buttons is 1px :D